### PR TITLE
MTDSA-17359: Updated changelog for Update the OAS description for periodData in Create, Amend, Retrieve CIS

### PR DESCRIPTION
**Pull request by darshanbala-hmrc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ to your Github account:
 
 You should now receive an email whenever the changelog is updated.
 
+---
+### 22 June 2023
+The following changes were deployed into sandbox and production:
```
Comment (mattstephens-hmrc) [inline README.md:42]: Have these actually been released i.e. made available in production?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,17 @@ to your Github account:
...
+
+#### cis-deductions-api
+
+* Updated `periodData` field description in API documentation for:
```
Comment (mattstephens-hmrc) [inline README.md:46]: can you add a "the" after "updated" and "in"?

File: README.md [hunk 3]
```diff
@@ -37,6 +37,17 @@ to your Github account:
...
+  * `Create CIS Deductions for Subcontractor`
+  * `Amend CIS Deductions for Subcontractor`
+  * `Retrieve CIS Deductions for subcontractor.`
```
Comment (mattstephens-hmrc) [inline README.md:49]: "Subcontractor"

Comment (mattstephens-hmrc) [reply README.md:49]: also shouldn't have the full-stop

File: README.md (hunk 1 above)
Comment (darshanbala-hmrc) [reply README.md:42]: @mattstephens-hmrc Yes they have been deployed to both ET and Production

**Review by mattstephens-hmrc — APPROVED** [review]

**Pull request merged**